email_mailbox/src/containers/Settings.js

```diff
@@ -141,11 +135,6 @@ const mapDispatchToProps = dispatch => {
     onUpdateLabel: params => {
       dispatch(updateLabel(params));
     },
-    onUploadAvatar: async params => {
```

Comment:
why it has been deleted?

Reply:
Itś repeated on `SettingsGeneralProfile` container